<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqDisplayProxyEditor</class>
 <widget class="QWidget" name="pqDisplayProxyEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ViewGroup" >
     <property name="title" >
      <string>View</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ViewData" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Toggle visibility of this dataset's geometry.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ViewZoomToData" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Change the camera location to best fit the dataset in the view window.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Zoom To Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ColorGroup" >
     <property name="title" >
      <string>Color</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QComboBox" name="ColorBy" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select method for coloring dataset geometry.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Color by:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="ColorMapScalars" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Map scalars through lookup table before applying color.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Map Scalars</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="ColorInterpolateColors" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Interpolate colors before mapping. Otherwise OpenGL is responsible of interpolation&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Interpolate Colors</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="pqColorChooserButton" name="ColorActorColor" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Edit the constant color for the geometry.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Set Solid Color...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Style</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Point size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Representation:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Interpolation:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Line width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="StyleRepresentation" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Choose what geometry should be used to represent the dataset.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="StyleInterpolation" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Choose the method used to shade the geometry and interpolate point attributes.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="StylePointSize" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If your dataset contains points/verticies, this scale adjusts the diameter of the rendered points.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>1</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="StyleLineWidth" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If your dataset containes lines/edges, this scale adjusts the width of the rendered lines.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>1</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Actor Control</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="OrientationX" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Orient the geometry relative to the dataset origin.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>180</double>
        </property>
        <property name="minimum" >
         <double>-180</double>
        </property>
        <property name="singleStep" >
         <double>5</double>
        </property>
        <property name="value" >
         <double>0</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QDoubleSpinBox" name="OriginY" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Set the origin point about which rotations take place.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="ScaleY" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Scale the geometry relative to the size of the dataset.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="ScaleX" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Scale the geometry relative to the size of the dataset.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="Opacity" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Set the opacity of the dataset's geometry.  Artifacts may appear in translucent geometry because primatives are not sorted.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>1</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QDoubleSpinBox" name="TranslateY" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Translate the geometry relative to the dataset location.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QDoubleSpinBox" name="TranslateZ" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Translate the geometry relative to the dataset location.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="OriginX" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Set the origin point about which rotations take place.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="ScaleZ" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Scale the geometry relative to the size of the dataset.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Origin:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="TranslateX" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Translate the geometry relative to the dataset location.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QDoubleSpinBox" name="OriginZ" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Set the origin point about which rotations take place.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>100</double>
        </property>
        <property name="minimum" >
         <double>-100</double>
        </property>
        <property name="singleStep" >
         <double>0.1</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Translate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Orientation:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Scale:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QDoubleSpinBox" name="OrientationY" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Orient the geometry relative to the dataset origin.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>180</double>
        </property>
        <property name="minimum" >
         <double>-180</double>
        </property>
        <property name="singleStep" >
         <double>5</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QDoubleSpinBox" name="OrientationZ" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Orient the geometry relative to the dataset origin.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="maximum" >
         <double>180</double>
        </property>
        <property name="minimum" >
         <double>-180</double>
        </property>
        <property name="singleStep" >
         <double>5</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="DismissButton" >
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Close the dialog&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>Dismiss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QPushButton</extends>
   <header>pqColorChooserButton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ViewData</tabstop>
  <tabstop>ViewZoomToData</tabstop>
  <tabstop>ColorBy</tabstop>
  <tabstop>ColorInterpolateColors</tabstop>
  <tabstop>ColorActorColor</tabstop>
  <tabstop>StyleRepresentation</tabstop>
  <tabstop>StyleInterpolation</tabstop>
  <tabstop>StylePointSize</tabstop>
  <tabstop>StyleLineWidth</tabstop>
  <tabstop>TranslateX</tabstop>
  <tabstop>TranslateY</tabstop>
  <tabstop>TranslateZ</tabstop>
  <tabstop>ScaleX</tabstop>
  <tabstop>ScaleY</tabstop>
  <tabstop>ScaleZ</tabstop>
  <tabstop>OrientationX</tabstop>
  <tabstop>OrientationY</tabstop>
  <tabstop>OrientationZ</tabstop>
  <tabstop>OriginX</tabstop>
  <tabstop>OriginY</tabstop>
  <tabstop>OriginZ</tabstop>
  <tabstop>Opacity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
